### PR TITLE
Add new + missing endpoints to project assets

### PR DESCRIPTION
**Pull request by cherry-john**
By submitting a PR for this repository you accept the contributor license agreement. 

### Description
Documents adam-rms/adam-rms#283

### Checklist

- [x] I have added documentation for new/changed functionality in this PR or in the documentation repo
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used, if not `main`

### CODE REVIEW

File: docs/api-v1-docs/projects/project-assets.md [hunk 1]
```diff
@@ -116,6 +116,37 @@ locationType
...
+
+ **Parameters**
+
+Parameters are POST unless otherwise noted
+
+```
+projects_id
+```
+
+## swap
```
Comment (Jbithell) [inline docs/api-v1-docs/projects/project-assets.md:134]: Is this correct?

**Review by Jbithell — CHANGES_REQUESTED** [review]

**Comment by Jbithell** [comment]
If we remove the #swap section I'll approve

File: docs/api-v1-docs/projects/project-assets.md (hunk 1 above)
Comment (cherry-john) [reply docs/api-v1-docs/projects/project-assets.md:134]: the swap endpoint was documented in the file, but not copied over to the the documentation

Comment (Jbithell) [reply docs/api-v1-docs/projects/project-assets.md:134]: I think this file talks about status list twice

Comment (cherry-john) [reply docs/api-v1-docs/projects/project-assets.md:134]: Have updated! That'll teach me to copy and paste wrong!

**Review by Jbithell — APPROVED** [review]

**Pull request merged**